lib/airbrake/logger/logger_ext.rb

```diff
@@ -75,4 +75,20 @@ def airbrake_severity(severity)
        'critical'
      end).freeze
   end
+
+  # @!macro helper_method
+  #   @note We define this helper instead of using instance variable to avoid
+  #     Ruby's warnings about uninitialized ivars.
+
+  ##
+  # @macro helper_method
+  def current_airbrake_notifier
+    airbrake_notifier || Airbrake[:default]
+  end
+
+  ##
+  # @marcro helper_method
+  def current_airbrake_severity
```

Comment:
The classic way is to use `get_`. Or rename `airbrake_severity_level` -> `_airbrake_severity_level`, `current_airbrake_severity` -> `airbrake_severity_level`